src/BenchmarkDotNet.Core/Characteristics/CharacteristicHelper.cs

```diff
@@ -56,12 +56,18 @@ public static IReadOnlyList<Characteristic> GetThisTypeCharacteristics(Type jobM
 
         private static IReadOnlyList<Characteristic> GetThisTypeCharacteristicsCore(Type jobModeType)
         {
-            var fieldValues = jobModeType.GetTypeInfo()
+            var fieldValues = jobModeType
+#if !UAP
+                .GetTypeInfo()
```

Comment:
I think that you could create an extension method for Type called GetTypeInfo() and get rid of all the ifdefines around this call

sth like:
```cs
#if UAP
internal static Type GetTypeInfo(this Type type) => type;
#endiff
```